meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb

```diff
@@ -17,8 +17,7 @@ UPSTREAM_CHECK_COMMITS = "1"
 
 inherit go-mod systemd
 
-GOBUILDFLAGS += '-tags vpn -ldflags="-X ${GO_IMPORT}/internal.Commit=${SRCREV}"'
-GO_LINKSHARED = ""
+GO_LDFLAGS += '-tags vpn -ldflags="-X ${GO_IMPORT}/internal.Commit=${SRCREV}"'
```

Comment:
nice.

Reply:
Had to change in the end as that was resulting in 2 ldflags at the same build line, and which didn't work right.

Same thing now, but we're setting the variable based on the original value, instead of just appending (there is no common build flag yet that can be used for recipes to customize just one small piece).